### ISSUE TITLE
[Endless Google] Scroll bug

### ISSUE THREAD

**Issue by Draxcyn** [tumpio/gmscripts#10]
Script works but scrolls back to top of page upon clicking a google result

**Comment by joeytwiddle** [comment]
Yeah that bugs me too. But it is easy to disable it. Just comment out these lines, or remove them:

```js
    window.addEventListener("beforeunload", function () {
        window.scrollTo(0, 0);
    }, false);
```

I'm not sure what that code is useful for. It's obviously _intended_ to scroll the page to the top when leaving it!
